iwyu.cc

```diff
@@ -3997,6 +3999,22 @@ int main(int argc, char **argv) {
   std::unique_ptr<clang::CompilerInstance> compiler(CreateCompilerInstance(
       options_parser.clang_argc(), options_parser.clang_argv()));
   if (compiler) {
+    // Figure out the absolute path to the resource-dir, which might be relative
```

Comment:
This code looks like it belongs in `CreateCompilerInstance` (which incidentally looks like it's already handling the resource dir, maybe that code should be removed?)